scheduler/crates/api/src/event/create_event.rs

```diff
@@ -219,7 +219,7 @@ mod test {
             busy: false,
             calendar_id: calendar.id.clone(),
             user,
-            reminders: vec![],
```

Comment:
macros will slow down compile time, where there is an equivalent, it's better to use that one.